tests/util/mod.rs

```diff
@@ -27,7 +27,9 @@ pub fn create_clients(nb_clients: usize) -> Vec<JoinHandle<()>> {
     let mut clients = Vec::new();
     for _ in 0..nb_clients {
         let client = std::thread::spawn(|| {
-            let _ = std::net::TcpStream::connect("127.0.0.1:64850").unwrap();
+            let stream = std::net::TcpStream::connect("127.0.0.1:64850").unwrap();
+            sleep(Duration::from_secs(100));
```

Comment:
Really want to leave this here ?

Reply:
Yes I want to leave this so that it's the manager that close the connection

Reply:
Ok, LGTM then